src/scripts/update_funding_round.js

```diff
@@ -111,25 +111,50 @@ const main = async () => {
 
             let allProposals = await getProposalsSelectQuery(`{Round} = ${curRoundNumber}`)
             const tokenPrice = await getTokenPrice()
-            const maxGrantUSD = curRound.get('Max Grant USD')
+            const basisCurrency = curRound.get('Basis Currency')
 
-            let earmarkedStructureWithOCEAN = JSON.stringify(addOCEANValueToEarmarks(curRound, tokenPrice))
+            let maxGrant = 0
+            let fundingAvailable = 0
 
-            const fundingAvailableUSD = curRound.get('Funding Available USD')
+            let maxGrantUSD = 0
+            let fundingAvailableUSD = 0
 
-            const maxGrantOCEAN = maxGrantUSD / tokenPrice
-            const fundingAvailableOCEAN = fundingAvailableUSD / tokenPrice
+            switch(basisCurrency) {
+                case 'USD' :
+                maxGrantUSD = curRound.get('Max Grant USD')
+                fundingAvailableUSD = curRound.get('Funding Available USD')
 
-            // Enter Due Diligence period
-            const roundUpdate = [{
+                maxGrant = maxGrantUSD/ tokenPrice
+                fundingAvailable = fundingAvailableUSD / tokenPrice
+                break
+
+                case 'OCEAN': 
+                const maxGrantOCEAN = curRound.get('Max Grant')
+                const fundingAvailableOCEAN = curRound.get('Funding Available')
+
+                maxGrant = maxGrantOCEAN
+                fundingAvailable = fundingAvailableOCEAN 
+
+                maxGrantUSD = maxGrantOCEAN *  tokenPrice
+                fundingAvailableUSD = fundingAvailableOCEAN * tokenPrice               
+                break
+
+                default:
+                    console.log('No Basis Currency was selected for this round.')
+            }
+
+            let roundUpdate = [{
                 id: curRound['id'],
                 fields: {  
                     'Round State': RoundState.DueDiligence,
                     'Proposals': allProposals.length,
                     'OCEAN Price': tokenPrice,
-                    'Max Grant': maxGrantOCEAN,
-                    'Earmarks': earmarkedStructureWithOCEAN,
-                    'Funding Available': fundingAvailableOCEAN,
+                    'Max Grant': maxGrant,
+                    'Earmarks': JSON.stringify(completeEarstructuresValues(curRound, tokenPrice, basisCurrency)),
+                    'Funding Available': fundingAvailable,
+                    'Max Grant USD': maxGrantUSD,
+                    'Earmarked USD': earmarkedUSD,
```

Comment:
I think you should make some changes here, regarding the `Earmarked USD`

Reply:
yes, that should be removed, forgot to remove it when getting the new changes from main